Documentation/ExtensionArchitecture/ConfigurationFiles/Index.rst

```diff
@@ -4,9 +4,9 @@
 .. _extension-configuration-files:
 
 
-===================
-Configuration Files
-===================
+========================================================
+Configuration Files (ext_tables.php & ext_localconf.php)
```

Comment:
We have way more configuration files nowadays, inside `Configuration` Folder. Some are already listed at https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/ExtensionArchitecture/FilesAndLocations/Index.html, but most are not linked to corresponding sections. We should cross reference those information.